lerobot/common/robot_devices/robots/utils.py

```diff
@@ -1,6 +1,13 @@
 from typing import Protocol
 
 
+def get_arm_id(name, arm_type):
+    """Returns the string identifier of a robot arm. For instance, for a bimanual manipulator
+    like Aloha, it could be left_follower, right_follower, left_leader, or right_leader.
+    """
+    return f"{name}_{arm_type}"
```

Comment:
Yes, it is called twice at two very different places in the code. It's to avoid possible issues if you update one location and not the other.

We will eventually remove it by improving our design.